test/parallel/test-module-strip-types.js

```diff
@@ -0,0 +1,92 @@
+'use strict';
+
+const common = require('../common');
+if (!process.config.variables.node_use_amaro) common.skip('Requires Amaro');
+const assert = require('assert');
```

Comment:
Non-blocking: for consistency

```suggestion
const assert = require('node:assert');
```